### DISCUSSION TITLE
-Discord members verifier to Avoid duplicate votes/ fake voters

### DISCUSSION THREAD

**Discussion by toney010319** [Group-1-hackathon/voting-system#3] [Q&A]
How can we ensure that there are no duplicate or fake votes? ie; si Andite may multiple accounts so paano yun?

**Comment by Raearn** [comment]
1. We create a discord bot account that scrapes the ID of every discord member joined in the server.
2. Integrate a 'Sign in using Discord' in our registration page.
3. Once signed in, a script should grab their discord ID and map/find it in the list of IDs scraped by the bot, and returns true or false.
4. If true, the user can now vote.

Let's take https://top.gg/ 's voting system as a reference.

**Reply by toney010319** [reply]
@gabrielleabasolo  @devpapi0891  @wilmer162000
